spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/storage/GcpStorageProperties.java

```diff
@@ -31,11 +31,22 @@
 public class GcpStorageProperties extends GoogleStorageProtocolResolverSettings implements
 		CredentialsSupplier {
 
+	/** The default bucket that should be used. Useful for environments like Cloud Foundry. */
```

Comment:
Why do you say so?
It is valid Javadoc format, and I can't find anything in the [Spring style guide](https://github.com/spring-projects/spring-framework/wiki/Code-Style) that says otherwise..

Reply:
M-m-m. 
At least this is not consistent what you have so far in other similar new properties in this PR...

Reply:
+1 with @artembilan 

Also we consistently do not start configuration keys with "The", "A". It would be nice if you follow the same consistency (or any consistency that is reasonable).

Reply:
May you confirm that this JavaDocs really becomes as a proper description hint for the configuration properties assistant?
Thanks